easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.6-foss-2017b-METIS-5.1.0.eb

```diff
@@ -0,0 +1,20 @@
+name = 'SuiteSparse'
+version = '4.5.6'
```

Comment:
@hajgato Did you try with the latest 5.1.2?

Reply:
I would expect that's just because it hasn't been updated for 2 years...

I don't care too much in this case because it's an old toolchain generation, but for newer toolchain we should really check whether SuiteSparse 5.1.2 works (I haven't seen problems with using that latest version in other places).

Reply:
Anyway, the toolchain `SuiteSparse` is 4.5.5

Reply:
So strictly speaking I should have used `4.5.5`

Reply:
There's actually no `SuiteSparse` at all yet for `foss/2017b` in `develop`.

Let's leave it as is, we can revisit this for newer toolchains.